Applications/DataExplorer/mainwindow.cpp

```diff
@@ -425,15 +425,18 @@ void MainWindow::save()
     }
     else if (fi.suffix().toLower() == "geo")
     {
-        int const return_val =
-            FileIO::GMSHInterface::writeGeoFile(_project.getGEOObjects(), fileName.toStdString());
+        std::vector<std::string> selected_geometries;
+        _project.getGEOObjects().getGeometryNames(selected_geometries);
 
-        if (return_val == 1)
+        GMSHInterface gmsh_io(
+            _project.getGEOObjects(), true,
+            FileIO::GMSH::MeshDensityAlgorithm::AdaptiveMeshDensity,
+            0.05, 0.5, 2, selected_geometries);
```

Comment:
Okay. It wasn't obvious before. It's okay for debugging geo-files but here should probably be an option to adjust these values. Maybe later....